pintpandas/pint_array.py

```diff
@@ -300,7 +301,7 @@ def astype(self, dtype, copy=True):
         array : ndarray
             NumPy ndarray with 'dtype' for its dtype.
         """
-        if isinstance(dtype,str) and (dtype.startswith("Pint[") or dtype.startswith("pint[")):
+        if isinstance(dtype,string_types) and (dtype.startswith("Pint[") or dtype.startswith("pint[")):
```

Comment:
does pint have a policy about linting/style nitpicks?  if so, a space after the comma here would be good.